src/screens/WelcomeScreen/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
```

Comment:
I like more putting an index file with
```
export { default } from './HomeScreen'
```
and then having `HomeScreen.tsx` and `HomeScreen.styles.ts`

So there is not a hundred files named the same, it makes files easier to find. WDYT?

Reply:
yes and also it is easier to find component files if they are not all named index

Reply:
Right. On the other side we'll have to create one more file for every new component that is redundant 🤔.

Reply:
Yup, I agree that neither solution is perfect but having worked with both I prefer much more the one that I am proposing, I think it also scales better in the developer experience aspect.
We can leave it up to the developer as well, if the developer is opinionated enough on the matter they will have no problem changing it. If they aren't they will use what's in the base.

Reply:
maybe we could bring this discussion on the slack channel and use a poll